components/org.wso2.identity.event.websubhub.publisher/src/main/java/org/wso2/identity/event/websubhub/publisher/service/WebSubEventPublisherImpl.java

```diff
@@ -213,14 +162,14 @@ private void handleTopicMgtResponse(CloseableHttpResponse response, HttpPost htt
             WebSubHubCorrelationLogUtils.triggerCorrelationLogForResponse(httpPost, requestStartTime,
                     WebSubHubCorrelationLogUtils.RequestStatus.COMPLETED.getStatus(),
                     String.valueOf(responseCode), responsePhrase);
-            handleSuccessfulTopicMgt(entity, topic, operation);
+            handleSuccessfulOperation(entity, topic, operation);
         } else if ((responseCode == HttpStatus.SC_CONFLICT && operation.equals(REGISTER)) ||
                 (responseCode == HttpStatus.SC_NOT_FOUND && operation.equals(DEREGISTER))) {
```

Comment:
what about 500 errors. Those should be captured as well

Reply:
Also, I don't think we should handle these error separately from unauthorized or forbidden. All these are server errors and we can mention the status and the error and fail the overall flow

Reply:
Address via https://github.com/wso2/product-is/issues/24085